tests/lcs/core/engine/_extension.lcb

```diff
@@ -33,4 +33,19 @@ public handler TestExtensionSupportModule_Handler()
 	return SupportHandler() is "support handler"
 end handler
 
+public handler TestExtensionExecuteScript_Do(in pScript as String, in pTarget as String, in pParams as List)
+	if pParams is the empty list and pTarget is the empty string then
```

Comment:
If I remember correctly, this can just be:

```
if pParams is empty and pTarget is empty then
   -- statements
end if
```

Yay for generics!  (Don't change this, it's just an observation)